test/test_rpc.py

```diff
@@ -1,17 +1,40 @@
+#!/usr/bin/env python3
+
 import sys
+import six
+import unittest
 
 import torch
 import torch.distributed as dist
 
 from common_distributed import MultiProcessTestCase
 from common_utils import load_tests, run_tests
 
+def my_function(a, b, c):
+    return a + b + c
+
+def no_result():
+    print("do nothing")
+
+class my_class:
+    def __init__(self, a):
+        self.a = a
+
+    def my_instance_method(self, b):
+        return self.a + b
+
+    @classmethod
+    def my_class_method(cls, d, e):
+        return d + e
+
+    @staticmethod
+    def my_static_method(f):
+        return f > 10
```

Comment:
nit: It would be clearer in the tests if the functions here also gave an indication of what they did (since you rely on the implementation details of these functions to compute what the correct test results are.)